ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -59,14 +72,34 @@ function BlockaidBannerAlert({ txData, ...props }) {
 
   const t = useContext(I18nContext);
   const { updateTransactionEventFragment } = useTransactionEventFragment();
+  const [isLoadingDisplayed, setIsLoadingDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (securityAlertResponse?.reason === 'loading') {
+      setIsLoadingDisplayed(true);
+    }
+  }, [securityAlertResponse]);
 
   if (
     !securityAlertResponse ||
     Object.keys(securityAlertResponse).length === 0
   ) {
     return null;
   } else if (securityAlertResponse.reason === 'loading') {
```

Comment:
The above 2 conditions are not same, loader should display only for duration when `securityAlertResponse.reason` is `loading`.

Reply:
oh... I think I see now. When I initially reviewed this PR I thought isLoadingDisplayed is meant to represent what you mentioned 
> duration when securityAlertResponse.reason is loading

maybe we can rename `isLoadingDisplayed` to `hasDisplayedLoading` or `hasRunLoading` because prepending with "is" implies this is the current state

Reply:
I did the renaming